packages/account/tests/account.e2e.spec.ts

```diff
@@ -1,9 +1,15 @@
 import { TestData } from "../../../tests";
-import { PaymasterUserOperationDto, createSmartAccountClient } from "../src/index";
+import {
+  PaymasterUserOperationDto,
+  createSmartAccountClient,
+  FeeQuotesOrDataResponse,
+  IHybridPaymaster,
+  PaymasterFeeQuote,
+  PaymasterMode,
+} from "../src/index";
```

Comment:
we moved form paymaster package to this package?

Reply:
We did this when we abstracted away the paymaster -> when we build it from the biconomyPaymasterApiKey

Reply:
I see. confused because I thought it was already merged in v4

Reply:
These are just some additional types / interfaces coming from the same Paymaster package. If we've made the decision to import that Paymaster class already to help with defaulting, then re-exporting both it and its relevant types and interfaces from accounts makes sense, so there's a central package to import stuff from. 

v4 has moved us in the direction of having only two user-facing packages: accounts + modules. Everything else will  just be imported into those packages.